core-scroll-threshold.html

```diff
@@ -153,21 +172,37 @@
       }
       if (!this.lowerThreshold) {
         this.lowerTriggered = false;
-      }      
+      }
     },
 
     checkThreshold: function(e) {
       var top = this._scrollTarget[this.scrollPosition];
-      if (!this.upperTriggered && this.upperThreshold != null) {
+      if (!this.upperTriggered && this.upperThreshold !== null) {
         if (top < this.upperThreshold) {
+
+          // No longer fire scroll events if there's no lower threshold or it
+          // has been triggered.
+          if (this.lowerThreshold === null ||
```

Comment:
this conditional can be just `if (this.lowerTriggered || this.lowerThreshold === null) {`